pyaedt/modules/Material.py

```diff
@@ -1992,16 +1993,248 @@ def set_magnetic_coercivity(self, value=0, x=1, y=0, z=0):
+        elif len(points_list_at_freq.keys()) > 1:
+            props["CoreLossMultiCurveData"] = OrderedDict({})
+            props["CoreLossMultiCurveData"]["property_data"] = "coreloss_multi_curve_data"
+            props["CoreLossMultiCurveData"]["coreloss_unit"] = "w_per_cubic_meter"
+
+            props["CoreLossMultiCurveData"]["AllCurves"] = OrderedDict({})
+            props["CoreLossMultiCurveData"]["AllCurves"]["OneCurve"] = []
+            for freq in list(points_list_at_freq.keys()):
+                points = list(itertools.chain(*[v for v in points_list_at_freq[freq]]))
+                one_curve = OrderedDict(
+                    {
+                        "Frequency": "{}Hz".format(freq),
+                        "Coordinates": OrderedDict({"DimUnits": ["", ""], "Points": points}),
+                    }
+                )
+                props["CoreLossMultiCurveData"]["AllCurves"]["OneCurve"].append(one_curve)
+
+        props = args = self._get_args(props)
+        props.pop(0)
+        coefficients = self.odefinition_manager.ComputeCoreLossCoefficients(
+            core_loss_model_type, self.mass_density.evaluated_value, props[0]
+        )
+        return coefficients
+
+    @pyaedt_function_handler()
+    def set_electrical_steel_coreloss_at_frequency(
+        self,
+        kdc=0,
+        cut_depth="1mm",
+        thickness="0.5mm",
+        conductivity=0,
+        coefficient_setup="w_per_cubic_meter",
+        core_loss_model_type="Electrical Steel",
+        points_list_at_freq={},
+    ):
+        """Set Electrical Steel Core Loss Model at one single frequency or at multiple frequencies.
 
         Parameters
         ----------
-        kh : float
-        kc : float
-        ke : float
         kdc : float
-        cut_depth : float
+            Coefficient considering the DC flux bias effects
+        cut_depth : str
+            Equivalent cut depth.
+            The manufacturing effects on core loss computation can be considered by using this parameter.
+            The default value is ``1mm``.
+        thickness : str
+            Thickness specified in terms of value + unit.
+            The default value is ``0.5mm``.
+        conductivity : float
+            Conductivity. The unit is S/m.
+            The default value is ``0S/m``.
+        coefficient_setup : str, optional
+            Core loss unit.
+            Possible values are: ``w_per_cubic_meter``, ``kw_per_cubic_meter``, ``w_per_lb``, ``w_per_kg``.
+            The default value is ``w_per_cubic_meter``.
+        points_list_at_freq : dict
+            Dictionary where keys are the frequencies (in Hz) and values are lists of points (BP curve).
+            If the core loss model is calculated at one frequency points_list_at_freq has to be provided as a
+            dictionary with one key (single frequency in Hz) and values are lists of points at
+            that specific freq (BP curve).
```

Comment:
```suggestion
            The default value is ``"1mm"``.
        thickness : str
            Thickness specified in terms of the value plus the unit.
            The default is ``"0.5mm"``.
        conductivity : float
            Conductivity. The unit is S/m.
            The default is ``"0S/m"``.
        coefficient_setup : str, optional
            Core loss unit. The default is ``"w_per_cubic_meter"``.
            Options are ``"kw_per_cubic_meter"``, ``"w_per_cubic_meter"``, ``"w_per_kg"``,
            and ``"w_per_lb"``.
        points_list_at_freq : dict
            Dictionary where keys are the frequencies (in Hz) and values are lists of points (BP curve).
            If the core loss model is calculated at one frequency, this parameter must be provided as a
            dictionary with one key (single frequency in Hz) and values are lists of points at
            that specific frequency (BP curve).
```
Should this description and the description for the same parameter earlier be the same?